newIDE/app/src/EventsSheet/EventsSearcher.js

```diff
@@ -11,6 +11,7 @@ export type SearchInEventsInputs = {|
   matchCase: boolean,
   searchInConditions: boolean,
   searchInActions: boolean,
+  searchInComments: boolean,
```

Comment:
Let's rename searchInComments to "searchInEventStrings"
to be consistent with the naming of the parameter in GDevelop Core :)

Reply:
arg missing this one